threshold_1: 0.95
threshold_2: 0.9
entries:
  - msg_type: sensor_msgs/CompressedImage
    topics:
      - topic: /main_camera/image_raw/compressed
        alias: downward
        target_frequency: 30.0
      - topic: /cam1/color/image_raw/compressed
        alias: forward
        target_frequency: 30.0
      - topic: /cam1/infra1/image_rect_raw/compressed
        alias: infra1
        target_frequency: 30.0
      - topic: /cam1/infra2/image_rect_raw/compressed
        alias: infra2
        target_frequency: 30.0
  - msg_type: sensor_msgs/Imu
    topics:
      - topic: /cam1/imu
        alias: RS IMU
        target_frequency: 200.0
      - topic: /mavros/imu/data
        alias: PX4 IMU
        target_frequency: 50.0
  - msg_type: sensor_msgs/NavSatFix
    topics:
      - topic: /mavros/global_position/raw/fix
        alias: gps_fix
        target_frequency: 5.0
  - msg_type: nav_msgs/Odometry
    topics:
      - topic: /odom_transform_nodelet/odomBinworld_from_transform
        alias: VIO
        target_frequency: 100.0
      - topic: /mavros/local_position/odom
        alias: local_position
        target_frequency: 30.0
  - msg_type: geometry_msgs/TwistStamped
    topics:
      - topic: /mavros/global_position/raw/gps_vel
        alias: global_position
        target_frequency: 8.0
